kratos/includes/kratos_application.h

```diff
@@ -128,7 +128,12 @@ class KRATOS_API(KRATOS_CORE) KratosApplication {
           mpModelers(rOther.mpModelers) {}
 
     /// Destructor.
-    virtual ~KratosApplication() {}
+    virtual ~KratosApplication() 
+    {
+        // This must be commented until tests have been fixed.
+        // DeregisterCommonComponents();
+        // DeregisterApplication();
```

Comment:
Should we keep this still commented?

Reply:
Yes, this is necessary and will be de-comented in the tests PR. This is because trying to de-regeister now (even if the code is correct) would cause some tests from Geo to fail as they try to unload the kernel but is never reload again (because of the python interface, kernel can only be loaded once even for cpp tests)